src/entries/popup/hooks/useUserAssetsBalance.ts

```diff
@@ -64,14 +64,16 @@ export function useUserAssetsBalance() {
     },
   );
 
-  const totalAssetsBalance = add(
-    totalAssetsBalanceKnownNetworks as string,
-    totalAssetsBalanceCustomNetworks as string,
-  );
+  const totalAssetsBalance =
+    totalAssetsBalanceKnownNetworks && totalAssetsBalanceCustomNetworks
+      ? add(totalAssetsBalanceKnownNetworks, totalAssetsBalanceCustomNetworks)
```

Comment:
Was the `as string` here legacy, or was there a particular reason we had to convert that type?

Reply:
It was type casted because of the undefined we had. Now we don't need to type cast it.

Reply:
Gotcha